pkg/domain/entities/images.go

```diff
@@ -305,8 +305,6 @@ type ImageSaveOptions struct {
 	OciAcceptUncompressedLayers bool
 	// Output - write image to the specified path.
 	Output string
-	// Do not save the signature from the source image
-	RemoveSignatures bool
```

Comment:
If this is the HTTP API, is it OK to change it like this? I guess unrecognized fields don’t cause failures?

Reply:
Thanks for checking. It's an internal (non-REST) API.  The REST handlers always set it to `true`.

Reply:
My mistake, I should write down how things work one of these days. It’s `pkg/domain/infra/tunnel.ImageEngine.Save(entities.ImageSaveOptions)`→`pkg/bindings/images.Export(pkg/bindings/images.ExportOptions)`→`pkg/api/handlers/libpod.ExportImages`→`pkg/domain/infra/abi.ImageEngine.Save(entities.ImageSaveOptions`, right?

Reply:
Yes, that is the exact flow.